metabase/metabase
# use the correct endpoint for downloading results from dashboards

Closes https://github.com/metabase/metabase/issues/28876

### Description

Switch to the correct endpoint when downloading results from dashcards. It should have been `api/dashboard/${dashboardId}/dashcard/${dashcardId}/card/${card.id}/query/${type}` instead of `api/card/${card.id}/query/${type}` which was used previously.

### How to verify

1. New question -> Sample Dataset -> Orders -> Save question
2. Create a dashboard, add the question
3. Ensure you can export csv/xlsx/json from the card
4. Add a dashboard filter, connect it to the question, filter by any value
5. Ensure you can export csv/xlsx/json from the card

### Checklist

- [x] Tests have been added/updated to cover changes in this PR — added e2e tests, will add specs on `utils.ts` later today
